platform/mellanox/mlnx-platform-api/sonic_platform/sfp_event.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
will SDK API called inside this function will also return the module id as an extra?

Reply:
Yes. But we don't need to handle them